make/langtools/netbeans/nb-javac/src/META-INF/upgrade/nbjavac.hint

```diff
@@ -105,6 +105,18 @@ $1.isEmpty() :: $1 instanceof java.util.Optional
 !$1.isPresent()
 ;;
 
+//TODO:
```

Comment:
- what will happen with this TODO?
- am I right this is a problem in Jackpot tool?
- is there a NetBeans/Jackpot issue reported to refer to?

Reply:
I've filled:
https://github.com/apache/netbeans-jackpot30/issues/38

What will happen is that either the problem in Jackpot will be fixed, or we drop support for JDK 8 here, and this rule will go away. Whichever will happen first.

Reply:
for the curious, I toyed around with a branch which builds nb-javac with JDK 17 bytecode level:
https://github.com/mbien/nb-javac/tree/target-jdk17-2

Reply:
Following notes seems to trigger a warning in my head...

> drop support for JDK 8
> toyed around with a branch which builds nb-javac with JDK 17 bytecode level

...looks like I will be being forced to accept your decisions soon... expect push back.

Reply:
> I've filled: [apache/netbeans-jackpot30#38](https://github.com/apache/netbeans-jackpot30/issues/38)

Referenced in 3f4ea60bfea6ba9e2f09230c5ee2cc0640ca8564